.github/workflows/ci.yml

```
@@ -97,6 +97,22 @@ jobs:
             }}
           token: ${{ secrets.CODECOV_TOKEN }}
 
+      - name: Upload awx jUnit test reports
+        if: >-
+          matrix.test.name == 'api-test'
+          && steps.make-run.outputs.test-result-files != ''
+          && github.event_name == 'push'
+          && github.repository == 'ansible/awx'
+          && github.ref_name == 'devel'
```

This is more reusable:
```suggestion
          && github.ref_name == github.event.repository.default_branch
```